source/agora/consensus/data/UTXOSet.d

```diff
@@ -381,9 +430,10 @@ private class UTXODB
         serializePart(value, dg);
 
         scope (failure) assert(0);
+        auto pubkey_hash = hashFull(value.output.address);
```

Comment:
If you just store the public key instead of the hash of the public key, then you wouldn't need to do a hash operation on a public key every time a lookup is made. Also, a public key is 32 bytes, the hash is 64 bytes. So we would save on memory use as well.

Reply:
Do you think it is a right approach to serialize and deserialize a public key?